<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateArcClothoidArcMeth2</class>
 <widget class="QDialog" name="CreateArcClothoidArcMeth2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create-Arc-Clothoid-Arc</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="6">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="radioButton_delta2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>0</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>inserted just delta 2</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_delta1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>inserted just delta 1</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>m</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QLineEdit" name="lineEdit_delta2">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="7" column="5">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>gon</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>gon               phi2=</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="lineEdit_phi1">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>clothoid and arc 1. Delta 2 is analog for arc 2.  Insert either delta 1 or delta 2 but not both. Clothoid</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEdit_R1">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Just draw a line. First point for arc-center 1 and second point for arc-center 2. Delta 1 starts</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>gon           delta 2=</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>m                     R2=</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>R1=</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QLineEdit" name="lineEdit_R2">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="13" column="4">
    <widget class="QPushButton" name="pushButtonGenerate">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>counterclockwise from positive x-axis to connection-line from arc-center1 to intersection between</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QLineEdit" name="lineEdit_phi2">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>delta1=</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>phi1=</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="lineEdit_delta1">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>gon</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string> is counterclockwise from arc 1 to arc 2, if R1&gt;R2. For clockwise clothoid swap everywhere 1 and 2.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
